backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/database/entity/AccessRecordEntity.kt

```diff
@@ -0,0 +1,13 @@
+package com.tngtech.apicenter.backend.connector.database.entity
+
+import org.hibernate.search.annotations.IndexedEmbedded
+import javax.persistence.EmbeddedId
+import javax.persistence.Entity
+
+@Entity
+data class AccessRecordEntity(
+        @EmbeddedId @IndexedEmbedded val accessRecordId: AccessRecordId,
+        val view: Boolean,
```

Comment:
Can't we model that as an `enum`, as each one implies the earlier ones? (I.e. `viewPrereleases` needs `view`, and `edit` needs `viewPrereleases`.)

Or can we somehow check these relations when setting them?